python_modules/libraries/dagster-dbt/dagster_dbt_tests/dbt_projects/test_dagster_metadata/packages.yml

```diff
@@ -1,4 +1,6 @@
 # We keep use `packages.yml` for compatability with `dbt-core==1.5.*`.
 # Once we remove support for that version, we should rename this file to `dependencies.yml`
 packages:
-  - local: "../../../dbt_packages/dagster"
+  - git: "https://github.com/dagster-io/dagster.git"
+    subdirectory: "python_modules/libraries/dagster-dbt/dbt_packages/dagster"
+    revision: 1.7.13
```

Comment:
```suggestion
    revision: master
```

Reply:
Ah yeah, I guess we can lock to a tag if we remove it 👍